integration-tests/emulated/src/mock/para_chains.rs

```diff
@@ -124,21 +118,19 @@ pub mod peregrine {
 decl_test_parachains! {
 	pub struct Spiritnet {
 		genesis = spiritnet::genesis(),
-		on_init = (),
-		runtime = {
-			Runtime: spiritnet_runtime::Runtime,
-			RuntimeOrigin: spiritnet_runtime::RuntimeOrigin,
-			RuntimeCall: spiritnet_runtime::RuntimeCall,
-			RuntimeEvent: spiritnet_runtime::RuntimeEvent,
+		on_init = {
+			spiritnet_runtime::AuraExt::on_initialize(1);
```

Comment:
Shouldn't this be `0`?

Reply:
In the examples in polkadot-SDK, they always used a `1`. I think it does not really matter

Reply:
No it's fine, I think it was just a typo.